### ISSUE TITLE
Can't easily get to my matches

### ISSUE THREAD

**Issue by mpricope** [theaigames/next#9]
Hey Guys,

The whole site is pretty fast lately, great job! However how can I see only my matches? The filter in "match log" is gone now :(

**Comment by mpricope** [comment]
Also something is weird on scoring:

These matches I won 
https://booking.riddles.io/competitions/hack-man/matches/c04accdd-4ff8-4c1d-ac4b-4e5f352cba51
https://booking.riddles.io/competitions/hack-man/matches/91dee72b-e5d6-4f79-880d-21276a37bdb3

But in the leaderboard I get -9 pts for each

**Comment by nikovanmeurs** [comment]
Yes, the filter has been temporarily disabled after last week's update. We changed quite a lot in the database schema and haven't had the time yet to update the "my matches" functionality to accommodate these changes.

We're out of office until Januari 2nd. We'll pick up both issues once we're back.

**Comment by JimVanEeden** [comment]
There was a problem with the ordering of the bots in the match log, which made it seem the wrong bot had gotten the points, but this was only a visual bug. The winner in the match was the correct one.

Fixed now, thanks for your report.

**Comment by mpricope** [comment]
Thanks guys! Just don't forget the "My Matches" filter :)
